vector/src/main/java/im/vector/app/features/home/room/detail/RoomDetailViewModel.kt

```diff
@@ -349,7 +352,13 @@ class RoomDetailViewModel @AssistedInject constructor(
             val roomId: String = room.roomId
             val confId = roomId.substring(1, roomId.indexOf(":") - 1) + widgetSessionId.toLowerCase(VectorLocale.applicationLocale)
 
-            val jitsiDomain = session.getHomeServerCapabilities().preferredJitsiDomain ?: stringProvider.getString(R.string.preferred_jitsi_domain)
+            val preferredJitsiDomain = tryThis {
+                awaitCallback<String> { rawService.getWellknown(session.myUserId, it) }
+                        .let { ElementWellKnownMapper.from(it) }
```

Comment:
This is used in multiple spaces, maybe we should provide an  extension method?

Reply:
Done